Monika After Story/game/script-grammar.rpy

```diff
@@ -0,0 +1,396 @@
+    m 3eub "Luckily, just like when we talked about '{b}who{/b}' versus '{b}whom{/b},' it turns out there's a simple way to figure out which one is correct!"
+    m 1euc "In our example, if you just take out '{b}[player] and{/b}' from the sentence, only one should make sense."
+    m 1hua "Let's try it out!"
+    m 3eud "The end result is: {w=0.5}'{b}I went on a date{/b}' or '{b}me went on a date{/b}.'"
+    m 3eub "Clearly, only the first one makes sense, so it's '{b}[player] and I went on a date{/b}.'"
+    m 1tuu "Oh, sorry, [player]...{w=1}did it make you feel left out when I said only '{b}I went on a date{/b}?'"
+    m 1hksdlb "Ahaha! Don't worry, I'd never leave you behind."
+    m 3eub "Now, on the other hand, if I was the object of the sentence, I would need to use '{b}and me{/b}' instead."
+    m 3eua "For example: {w=0.5}'{b}Natsuki asked [player] and me if we liked her cupcakes.{/b}'"
+    m 1eub "I hope that helps the next time you come across this situation while writing, [player]!"
+    m 3hub "Anyway, that's all for today, thanks for listening!"
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_gtod_tip009",
+            category=["grammar tips"],
+            prompt="Apostrophes",
+            conditional="store.mas_gtod.has_day_past_tip(8)",
+            action=EV_ACT_POOL
+        )
+    )
+
+# Note: formatted apostrophes have been used in examples in this tip for clarity. Please DO NOT remove them.
+label monika_gtod_tip009:
+    if player[-1].lower() == 's':
+        $ tempname = player
+    else:
+        $ tempname = 'Alexis'
+
+    m 1eua "Today we're going to talk about apostrophes. Pretty straightforward, right?"
+    m 3eua "Add them to show possession: '{b}Sayori’s fork, Natsuki’s spoon, Yuri’s knife{/b}...'"
+    m 1esd "I guess the issue that can come up is when you have to add an apostrophe to a word that ends with an '{b}s{/b}.'"
+    m 3eub "For plural words, this is simple; just add the apostrophe at the end: {w=0.5}'{b}monkeys’{/b}.'"
+    m 1hksdla "It's pretty clear that '{b}monkey’s{/b},' which would indicate possession belonging to a single monkey, or '{b}monkeys’s{/b}' would be wrong."
+    m 1eud "The gray area that comes up is when we bring in people's names, like '{b}Sanders{/b}' or '{b}[tempname]{/b}.'"
+    m 1euc "In some style guides I've read, it seems that we usually add an apostrophe and '{b}s{/b}' as usual, with the exception of historical names like '{b}Sophocles{/b}' or '{b}Zeus{/b}.'"
+    m 3eub "Personally, I think all that matters here is consistency!"
+    m 3esd "If you're going to go with '{b}[tempname]’{/b},' then it's fine as long as you use '{b}[tempname]’{/b}' for the entire text."
+    m 1tuu "That matters more than honoring some old Greeks to me."
+    m 3eud "One interesting exception is the case of '{b}its{/b}' versus '{b}it’s{/b}.'"
```

Comment:
Like with the conjunctions lead-in at the end of _Comma Splices and Run-ons_, I feel like this could lead into a whole new tip for _Its vs. It's_?

The explanation between the difference here is sound, but having it in a new tip could open more space for examples.